### DISCUSSION TITLE
Unable to pay and activate account

### DISCUSSION THREAD

**Discussion by myperfectfit-dev** [vercel/community#4506] [Help]
### Question

Hi,

We are having the pro account and trying to pay to the vercel its showing the error from past 7 days now the account is forzen and support not replying despite of sev -1 issue can any one help me how to get out of this. Despite of trying new card its throwing error.
Whats the best method to get this payment done.

Thanks,
Raj

**Comment by myperfectfit-dev** [comment]
[#00166188](https://vercel.com/teams/mpf-dev/support/cases/00166188) is my ticket number any one please help

**Reply by myperfectfit-dev** [reply]
Hi ,

I have got the email but the issue is still their. We need to add new card and then do the payment. Currently the email which you generated is not showing to pay any thing. Kindly give us a new link where i can add a card their by i can pay thigns.

**Comment by swarnava** [comment]
Hi,

We have reached out to you!

**Reply by myperfectfit-dev** [reply]
Hi ,

I have got the email but the issue is still their. We need to add new card and then do the payment. Currently the email which you generated is not showing to pay any thing. Kindly give us a new link where i can add a card their by i can pay thigns.

**Reply by myperfectfit-dev** [reply]
I am getting the following error 

This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

**Comment by myperfectfit-dev** [comment]
I am getting the following error

This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.
